sitewit/models.py

```diff
@@ -113,3 +119,30 @@ def delete(cls, account_token):
         result = cls.get_service().delete_account(account_token)
 
         return Account(result)
+
+
+class Subscription(SiteWitServiceModel):
+    def __init__(self, site_id, url, data):
+        self.site_id = site_id
+        self.url = url
+        self.ad_spend = Decimal(data['budget'])
+        self.price = Decimal(data['fee'])
+        self.campaign_id = data['campaignId']
+        self.currency = data['currency']
+
+    @classmethod
+    def iter_subscriptions(cls):
```

Comment:
I'm not sure I understand what you're saying here. Can you restate your objection? What @RayeN is doing here is becoming pretty standard in our service client models.